docker-build-env/Dockerfile

```diff
@@ -13,4 +13,21 @@ RUN apt-get update && DEBIAN_FRONTEND=noninteractive apt-get -y --no-install-rec
         ninja-build \
         pkg-config \
         wget \
+        # headers for p11 engine
+        # TODO: this may not be necessary if we don't use libp11 directly (only though openssl)
+        libp11-dev \
```

Comment:
Nit: Dependencies are in alphabetical order.

Reply:
Done.